packages/moonstone/ProgressBar/ProgressBar.js

```diff
@@ -106,7 +105,7 @@ const ProgressBarBase = kind({
 		 * @type {Boolean|Element|Function}
 		 * @public
 		 */
-		tooltip: PropTypes.oneOfType([PropTypes.bool, PropTypes.object, PropTypes.func])
+		tooltip: PropTypes.oneOfType([PropTypes.bool, PropTypes.element, PropTypes.func])
```

Comment:
I don't think we want to change this API.  I think we don't want 'object' or 'element' here...

Reply:
I will add `object`, but we want an element here.